cli/popper/utils.py

```diff
@@ -158,12 +158,14 @@ def b(t):
         try:
             out = check_output(cmd, shell=True, stderr=PIPE,
                                universal_newlines=True)
+            returncode = 0
```

Comment:
we can use `ecode` below here. That is, we can move the declaration of the `ecode` variable before this block of code and work with it instead of defining `returncode`